dev/intro.rst

```diff
@@ -150,15 +141,18 @@ lib/
 man/
    Manual pages, generated from the documentation.
 
-pkg/
-   Compiled packages, generated by the build process.
-
-protocol/
-   Legacy location of the protocol package.
-
 script/
    Various utility scripts for auto generating stuff and so on.
 
+  genassets/
+      Generates asset files that are compiled into ``syncthing`` as part of the build process (build utility).
+
+  transifexdl/
+      Downloads translations from Transifex (build utility).
```

Comment:
There seems to be some inconsistent indentation here, not sure how significant

Reply:
Fixed, thx.